src/twisted/names/test/test_names.py

```diff
@@ -1211,23 +1212,58 @@ class BindAuthorityTests(unittest.TestCase):
     Tests for L{twisted.names.authority.BindAuthority}.
     """
 
-    def loadBindString(self, s):
+    def loadBindString(self, s: bytes, asText: bool = True) -> authority.BindAuthority:
         """
         Create a new L{twisted.names.authority.BindAuthority} from C{s}.
 
         @param s: A string with BIND zone data.
-        @type s: bytes
+        @type s: L{bytes}
```

Comment:
FYI, these `@type` and `@rtype` fields are no longer required, as pydoctor can read the annotation directly and will document it properly. It's not worth removing (particularly in a test module) but in the future just avoid adding that redundant info, and remove them anywhere that they match exactly with a type annotation.